python/ray/tune/tune.py

```diff
@@ -35,39 +35,113 @@ def _make_scheduler(args):
             args.scheduler, _SCHEDULERS.keys()))
 
 
-def _find_checkpoint_dir(exp_list):
-    assert exp_list, "Experiments must be specified via `run_experiments`"
-    exp = exp_list[0]
-    # TODO(rliaw): Make sure this is resolved earlier.
+def _find_checkpoint_dir(exp):
+    # TODO(rliaw): Make sure the checkpoint_dir is resolved earlier.
```

Comment:
Is this a todo that should be resolved before merging this or just a note for future?

Reply:
Note for future (I just made it more concrete)